Procfile.example

```diff
@@ -49,6 +49,12 @@
 # vmware_event_<id>:        ruby lib/workers/bin/run_single_worker.rb --ems-id <id> ManageIQ::Providers::Vmware::InfraManager::EventCatcher
 # vmware_refresh_core_<id>: ruby lib/workers/bin/run_single_worker.rb --ems-id <id> MiqEmsRefreshCoreWorker
 
+# Embedded Ansible workers
+
+# ansible: ruby lib/workers/bin/run_single_worker.rb EmbeddedAnsibleWorker
```

Comment:
Should this be `embedded_ansible`?

Reply:
This is the first time I'm seeing this file, you tell me ;) Should it?

Reply:
Nope, this is good as is.

It should be the name of the worker class.